ddtrace/contrib/django/patch.py

```diff
@@ -401,60 +400,66 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
+                        elif isinstance(
+                            template,
+                            (
+                                list,
+                                tuple,
+                            ),
+                        ):
+                            template_names = template
+                        elif hasattr(template, "template"):
+                            # ^ checking by attribute here because
+                            # django backend implementations don't have a common base
+                            # `.template` is also the most consistent across django versions
+                            template_names = [template.template.name]
+                        else:
+                            template_names = None
+
+                        utils.set_tag_array(span, "django.response.template", template_names)
+
+                    url = utils.get_request_uri(request)
+
+                    if django.VERSION >= (2, 2, 0):
+                        request_headers = request.headers
                     else:
-                        template_names = None
-
-                    utils.set_tag_array(span, "django.response.template", template_names)
-
-                url = utils.get_request_uri(request)
+                        request_headers = {}
+                        for header, value in request.META.items():
+                            name = from_wsgi_header(header)
+                            if name:
+                                request_headers[name] = value
+
+                    response_headers = dict(response.items())
+                    trace_utils.set_http_meta(
+                        span,
+                        config.django,
+                        method=request.method,
+                        url=url,
+                        status_code=status,
+                        query=request.META.get("QUERY_STRING", None),
+                        request_headers=request_headers,
+                        response_headers=response_headers,
+                    )
 
-                if django.VERSION >= (2, 2, 0):
-                    request_headers = request.headers
-                else:
-                    request_headers = {}
-                    for header, value in request.META.items():
-                        name = from_wsgi_header(header)
-                        if name:
-                            request_headers[name] = value
-
-                response_headers = dict(response.items())
-                trace_utils.set_http_meta(
-                    span,
-                    config.django,
-                    method=request.method,
-                    url=url,
-                    status_code=status,
-                    query=request.META.get("QUERY_STRING", None),
-                    request_headers=request_headers,
-                    response_headers=response_headers,
-                )
+            except django.core.exceptions.ImproperlyConfigured:
```

Comment:
```suggestion
```

While this is one of the errors that could be raised I don't think we want to make any assumptions about it (eg. like if it's related to retrieving the authentication information for the user or not).